Race condition somewhere in build

When attempting to build CVMFS using the arch PKGBUILD published here:
https://aur.archlinux.org/packages/cvmfs

The build fails with the following error:
```
[ 27%] Generating man page for cvmfs_config
help2man: can't get `--help' info from /home/gdevenyi/projects/aur/cvmfs/src/cvmfs-cvmfs-2.13.3/build/cvmfs/cvmfs2
Try `--no-discard-stderr' if option outputs to stderr
make[2]: *** [doc/manpages/CMakeFiles/manpages.dir/build.make:74: doc/manpages/cvmfs2.1] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:674: doc/manpages/CMakeFiles/manpages.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 27%] Built target cvmfs_util_debug
[ 27%] Built target mount.cvmfs
[ 27%] Built target cvmfs_util
[ 27%] Built target cvmfs_fuse3_stub
make: *** [Makefile:136: all] Error 2
==> ERROR: A failure occurred in build().
    Aborting...
```

If I patch the PKGBUILD file to instead build with `make -j1` then the build is successful.

I believe that indicates there's a dependecy race condition somewhere in the Makefile/CMake config.

--- a/doc/manpages/CMakeLists.txt
+++ b/doc/manpages/CMakeLists.txt
@@ -24,7 +24,7 @@ else()
             --output=${MAN_OUTPUT_DIR}/cvmfs2.1
             --name "cvmfs2: fuse client for the CernVM-Filesystem" 
             "$<TARGET_FILE:cvmfs2>"
-        DEPENDS cvmfs2
+        DEPENDS cvmfs2 cvmfs_fuse3
         WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/cvmfs/
         COMMENT "Generating man page for cvmfs2"
         VERBATIM

--- a/doc/manpages/CMakeLists.txt
+++ b/doc/manpages/CMakeLists.txt
@@ -24,7 +24,7 @@ else()
             --output=${MAN_OUTPUT_DIR}/cvmfs2.1
             --name "cvmfs2: fuse client for the CernVM-Filesystem" 
             "$<TARGET_FILE:cvmfs2>"
-        DEPENDS cvmfs2 cvmfs_fuse3
+        DEPENDS cvmfs2 cvmfs_fuse3 cvmfs_fuse3_stub
         WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/cvmfs/
         COMMENT "Generating man page for cvmfs2"
         VERBATIM

